### ISSUE TITLE
rm: cannot remove '/usr/share/libalpm/hooks/package-cleanup.hook': No such file or directory

### ISSUE THREAD

**Issue by bitwave** [aji-prod/tomato#3]
while rebulding the image this error appears:
```
rm: cannot remove '/usr/share/libalpm/hooks/package-cleanup.hook': No such file or directory
```

**Comment by aji-prod** [comment]
Hi,

The commit 9580037d204c170d6fd43ef5daa0b430ac9e21db reintroduced the built-in `'/usr/share/libalpm/hooks/package-cleanup.hook` hook.
This hook is no more available from the official [docker archlinux image](https://hub.docker.com/_/archlinux/), this will be fixed with #2 

The original `/usr/share/libalpm/hooks/package-cleanup.hook` was https://github.com/archlinux/archlinux-docker/commit/601fcd25268889138b7228ce932416d59d11c7eb

**Comment by aji-prod** [comment]
Hi,

The release v0.7.0 (9c1df3f) is now based on the [archlinux](https://hub.docker.com/_/archlinux/) docker image - #2.
Which does no more include a `package-cleanup.hook`.
